listenbrainz/webserver/static/js/src/listens/ListenCard.tsx

```diff
@@ -264,8 +265,13 @@ export default class ListenCard extends React.Component<
                       <span
                         className="listen-time text-muted"
                         title={
-                          listen.listened_at_iso?.toString() ||
-                          new Date(listen.listened_at * 1000).toISOString()
+                          listen.listened_at
+                            ? fullLocalizedDateFromTimestampOrISODate(
+                                listen.listened_at * 1000
+                              )
+                            : fullLocalizedDateFromTimestampOrISODate(
+                                listen.listened_at_iso
+                              )
                         }
                       >
                         {preciseTimestamp(
```

Comment:
unrelated to this PR, but I noticed that the `...` dropdown on a listen card still has a title of "delete" (probably from when it was a delete button). Could this be updated? "more"? "tools" ?

Reply:
I went ahead and corrected that in this PR, might as well.